### PR TITLE
Update flatpak manifest

### PR DESCRIPTION
**Pull request by a-andre**

### CODE REVIEW

**Comment by OzarkShepherd** [comment]
Thank you so much for updating the dependencies. Since you know a way to remove the no-pep517 command, would you be able to add pygraphviz back in? https://github.com/pygraphviz/pygraphviz Pygraphviz is needed for one of the addons, I think network chart.

I see you saw that the own-name is necessary due to gramps using a hyphen instead of underscore, so does that mean you ran a workflow and have a gramps.flatpak file to test? I didn't see it at your repository, so would you add the file to the pull request after adding pygraphviz if possible?

**Comment by a-andre** [comment]
I can add pygraphviz but I don't know how to test as the network chart addon seems to be no longer available (I couldn't find it with the addon manager). 

Gramps no longer uses the hyphen (https://github.com/gramps-project/gramps/commit/2b50f55991f2e3025e888e8d4f1bcc930e60b1b2) that's why I removed the own-name arg.

**Comment by Nick-Hall** [comment]
> I can add pygraphviz but I don't know how to test as the network chart addon seems to be no longer available (I couldn't find it with the addon manager).

Try searching with the Audience set to "Expert".

**Comment by OzarkShepherd** [comment]
That's fine about pygraphviz, I don't actually know if network chart is no longer compatible with 5.2, or if it doesn't show up due to the flatpak not having the required pygraphviz prerequisite.

Would you add the gramps.flatpak you have from the Actions workflow? I just want to verify that the updates didn't break anything. I need to start adding the archive with the "Paste, drop, or click to add files" link also when I do PR's.

**Comment by OzarkShepherd** [comment]
I just noticed the 25 MB attachment limit makes my request unfeasible, so I'll check the archive soon another way.

**Comment by OzarkShepherd** [comment]
Thank you a-andre , I didn't notice any problems with the updated dependencies. Spell check works, edit exif still works, graphview still works.

So since it is not a bug fix (unlike the data directories I'm troubleshooting in another PR) I guess this can be merged on Saturday 16 March to give more time for others to review?

File: org.gramps_project.Gramps.yml [hunk 1]
```diff
@@ -154,15 +141,15 @@ modules:
         url:  https://gitlab.gnome.org/GNOME/geocode-glib/-/archive/master/geocode-glib-master.tar.gz
         sha256:  9ec6747d31fee70e5b7ede079f63402b67c21c329ade1d8a8ef9c76de5136901
```
Comment (OzarkShepherd) [inline org.gramps_project.Gramps.yml:142]: geocode glib master changed 2 days ago on March 16, which changed the sha256sum. So the sha256sum has to be changed here to compile and push this update for the Gramps flatpak.

**Review by OzarkShepherd — CHANGES_REQUESTED** [review]
geocode glib master changed upstream and needs a new sha256sum

File: org.gramps_project.Gramps.yml (hunk 1 above)
Comment (a-andre) [reply org.gramps_project.Gramps.yml]: I changed the URL to a permanent link

**Review by OzarkShepherd — APPROVED** [review]

**Pull request merged**